dev/Repeater/ItemsRepeater.idl

```diff
@@ -106,6 +106,9 @@ unsealed runtimeclass ItemsSourceView: Windows.UI.Xaml.Interop.INotifyCollection
     Boolean HasKeyIndexMapping{ get; };
     String KeyFromIndex(Int32 index);
     Int32 IndexFromKey(String key);
+
+    [WUXC_VERSION_PREVIEW]
+    Int32 IndexOf(IInspectable item);
```

Comment:
For Testing, you can create an api test in [https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/Repeater/APITests/InspectingDataSourceTests.cs](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/Repeater/APITests/InspectingDataSourceTests.cs) and just call IndexOf with different values.

Reply:
Good idea, added a new test there.